wal/checkpoint.go

```diff
@@ -146,14 +147,14 @@ func Checkpoint(w *wal.WAL, from, to int, keep func(id uint64) bool, mint int64)
 		os.RemoveAll(cpdirtmp)
 	}()
 
-	r := wal.NewReader(sgmReader)
+	r := NewReader(sgmReader)
 
 	var (
-		series  []RefSeries
-		samples []RefSample
-		tstones []Stone
-		dec     RecordDecoder
-		enc     RecordEncoder
+		series  []record.RefSeries
+		samples []record.RefSample
+		tstones []tombstones.Stone
+		dec     record.RecordDecoder
```

Comment:
Now that `RecordDecoder` and `RecordEncoder` are in their own package these stutter a bit. Perhaps rename to `record.Decoder` and `record.Encoder`?